reporter.go

```diff
@@ -195,7 +202,7 @@ func createJUnitReportSchema(runDetails *rainforest.RunDetails, api reporterAPI)
 
 			if test.Result == "failed" {
 				log.Printf("Fetching information for failed test #" + strconv.Itoa(test.ID))
-				testDetails, err := api.GetRunTestDetails(runDetails.ID, test.ID)
```

Comment:
I found this super confusing - as it clobbers the globally available `api` object. So I renamed it